streams/fr.m3u

```diff
@@ -179,3 +179,27 @@ https://tv3v.hdr-tv.com/live/tv3v/livestream/master.m3u8
 https://srv.webtvmanager.fr:3118/live/viatelepaeselive.m3u8
 #EXTINF:-1 tvg-id="DieuTV.fr",DieuTV (1080p)
 https://katapy.hs.llnwd.net/dieutvwza1/DIEUTVLIVE/smil:dieutv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Kosmo.fr" status="online",Kosmo FR
```

Comment:
For the future, if the channel ID is not taken from [iptv-org/database](https://github.com/iptv-org/database), you can not specify it at all.
```suggestion
#EXTINF:-1 tvg-id="" status="online",Kosmo FR
```